src/audacity.desktop.in

```diff
@@ -51,6 +57,8 @@ StartupWMClass=Audacity
 Type=Application
 Categories=AudioVideo;Audio;AudioVideoEditing;
 
+Keywords=sound;musicediting;voicechannel;frequency;modulation;audioclipping;;noisecancellation;multitrackaudioeditor:edit;mixing;WAV;AIFF;FLAC;MP2;MP3;
```

Comment:
you have a double `;;` in here as well as a `:` instead of `;`. I also don't understand some of your keyword choices here: Why audioclipping or noisecancellation?  

Further, you are allowed to use spaces in keywords.